### DISCUSSION TITLE
error undefine index

### DISCUSSION THREAD

**Discussion by sudfunk** [fx-project-OS/fx-project-Open-Source#6] [General]
Dear All ..
I have some problem in installation I Had so many Error Undefined index during and after installation
here these Error :
Notice: Undefined index: nl_b in C:\xampp\htdocs\fxpm\basics.inc on line 99

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 848

Notice: Undefined variable: los in C:\xampp\htdocs\fxpm\basics.inc on line 916

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1074

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1092

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1118

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1890

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1714

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 937

Notice: Undefined variable: locstoid in C:\xampp\htdocs\fxpm\basics.inc on line 946

Notice: Undefined variable: locstoid in C:\xampp\htdocs\fxpm\basics.inc on line 948

Notice: Undefined variable: sesstoid in C:\xampp\htdocs\fxpm\basics.inc on line 946

Notice: Undefined variable: sesstoid in C:\xampp\htdocs\fxpm\basics.inc on line 948

Notice: Undefined variable: locseskey in C:\xampp\htdocs\fxpm\basics.inc on line 946

Notice: Undefined variable: locseskey in C:\xampp\htdocs\fxpm\basics.inc on line 948

Notice: Undefined variable: locsescnt in C:\xampp\htdocs\fxpm\basics.inc on line 946

Notice: Undefined variable: locsescnt in C:\xampp\htdocs\fxpm\basics.inc on line 948

Notice: Undefined variable: emode in C:\xampp\htdocs\fxpm\basics.inc on line 946

Notice: Undefined variable: emode in C:\xampp\htdocs\fxpm\basics.inc on line 948

Notice: Undefined variable: lts in C:\xampp\htdocs\fxpm\basics.inc on line 946

Notice: Undefined variable: lts in C:\xampp\htdocs\fxpm\basics.inc on line 948

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1541

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1586

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1031

Notice: Undefined index: __headers in C:\xampp\htdocs\fxpm\basics.inc on line 382

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 2331

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 2842

Notice: Undefined index: client in C:\xampp\htdocs\fxpm\basics.inc on line 2851

Notice: Undefined index: person in C:\xampp\htdocs\fxpm\basics.inc on line 2855

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 1052

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 6612

Notice: Undefined index: cstyle in C:\xampp\htdocs\fxpm\basics.inc on line 509

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 4881

Notice: Undefined variable: fxfooter in C:\xampp\htdocs\fxpm\SYS\worker.inc on line 1005

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 4569

Notice: Undefined index: dbid in C:\xampp\htdocs\fxpm\SYS\worker.inc on line 164

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 2331

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 2842

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 104

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 6704

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 8363

Notice: Undefined index: calendar in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: curl in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: iconv in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: mbstring in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: openssl in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: sockets in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: gd in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: pgsql in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: sqlsrv in C:\xampp\htdocs\fxpm\basics.inc on line 8409

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\basics.inc on line 8464

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 5969

Notice: Undefined index: forceprotocol in C:\xampp\htdocs\fxpm\SYS\db.inc on line 5972

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 529

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 1198

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 821

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 6629

Notice: Undefined index: debugmode in C:\xampp\htdocs\fxpm\SYS\db.inc on line 6585

Notice: Undefined variable: tokenPos in C:\xampp\htdocs\fxpm\SYS\db.inc on line 6598

Notice: Undefined index: _maxlength in C:\xampp\htdocs\fxpm\SYS\db.inc on line 836
and more ...

my configuration  :
XAMPP for Windows  7.4.ss 64-bit
PostgreSQL 10 64-bit

please help me .. 🙏

**Comment by fx-project-OS** [comment]
Dear sudfunk,

these messages are "Notices" and not errors. You can simply disable the output of Notices by setting a different value in your "php.ini"
We suggest setting this value to only see "real" errors:

error_reporting = E_COMPILE_ERROR|E_RECOVERABLE_ERROR|E_ERROR|E_CORE_ERROR

You may have to restart your webservice (e.g. Apache) for this setting to take effect.
We hope this helps with your problem and that you can now enjoy fx-project OS.
